app/src/main/java/edu/rpi/shuttletracker/MapsActivity.kt

```diff
@@ -59,6 +59,8 @@ import kotlin.coroutines.*
 
 class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
 
+    var busMarkerArray: ArrayList<Marker> = ArrayList<Marker>()

```

Comment:
This should be a private variable before we go through with the pull request, because it means that it won't be accessible to other classes, which is generally good practice. This makes it clear to other programmers that they shouldn't access the bus marker array in other classes.